src/handlers/ens.ts

```diff
@@ -489,12 +489,22 @@ export const fetchAccountPrimary = async (accountAddress: string) => {
 };
 
 export function prefetchENSIntroData() {
-  for (const name of ensIntroMarqueeNames) {
-    prefetchENSAddress({ name }, { staleTime: Infinity });
-    prefetchENSAvatar(name, { cacheFirst: true });
-    prefetchENSCover(name, { cacheFirst: true });
-    prefetchENSRecords(name, { cacheFirst: true });
-    prefetchFirstTransactionTimestamp({ addressOrName: name });
+  const ensMarqueeQueryData = queryClient.getQueryData<{
+    ensMarquee: EnsMarqueeAccount[];
+  }>([ENS_MARQUEE_QUERY_KEY]);
+
+  const ensMarqueeAccounts = (
+    ensMarqueeQueryData?.ensMarquee || getEnsMarqueeFallback()
```

Comment:
If we truly want to have a fallback, then why not just return these fallback values from the main query fetcher?

Reply:
im new here, ty

Reply:
@estrattonbailey don't i need this anyways since there is a case where the data hasnt been fetched yet ?